lib/rock/packaging/templates/debian/control

```diff
@@ -2,32 +2,6 @@ Source: <%= debian_name %>
 Section: science
 Priority: extra
 Maintainer: Rock Packaging Daemon <rock-dev@dfki.de>
-% dependencies = (deps_rock_packages + deps_osdeps_packages + deps_nonnative_packages).flatten
```

Comment:
That was moved to generate_debian_dir. dependencies is only used to when generating build_dependencies.